Index: linux-4.14.81/lib/pci_iomap.c
===================================================================
--- linux-4.14.81.orig/lib/pci_iomap.c
+++ linux-4.14.81/lib/pci_iomap.c
@@ -33,17 +33,28 @@ void __iomem *pci_iomap_range(struct pci
 	resource_size_t len = pci_resource_len(dev, bar);
 	unsigned long flags = pci_resource_flags(dev, bar);
 
-	if (len <= offset || !start)
+	if (len <= offset) {
+		pr_warn("Length smaller or equal offset");
 		return NULL;
+	}
+
+	if (!start) {
+		pr_warn("No start");
+	}
 	len -= offset;
 	start += offset;
 	if (maxlen && len > maxlen)
 		len = maxlen;
-	if (flags & IORESOURCE_IO)
+	if (flags & IORESOURCE_IO) {
+		pr_warn("IORESSOURCE_IO");
 		return __pci_ioport_map(dev, start, len);
-	if (flags & IORESOURCE_MEM)
+	}
+	if (flags & IORESOURCE_MEM) {
+		pr_warn("IORESSOURCE_MEM");
 		return ioremap(start, len);
+	}
 	/* What? */
+	pr_warn("What?");
 	return NULL;
 }
 EXPORT_SYMBOL(pci_iomap_range);
